notebooks/azureml_model_headers_demo/german_credit_azure_ml_certifai_pro_demo.ipynb

```diff
@@ -0,0 +1,1130 @@
+   "source": [
+    "## Running Cortex Certifai Scan locally\n",
+    "\n",
+    "1. Install the cortex certifai packages required to initiate model scan\n",
+    "\n",
+    "2. Configure scan details and execute\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installing Cortex Certifai python packages\n",
+    "\n",
+    "initiating a Cortex Certifai scan requires following python packages to be installed in the current local environment\n",
+    "\n",
+    "`required-packages`\n",
+    "\n",
+    "- cortex-certifai-scanner\n",
+    "- cortex-certifai-engine\n",
+    "- cortex-certifai-common\n",
+    "\n",
+    "`optional-packages`\n",
+    "\n",
+    "- cortex-certifai-client\n",
+    "- cortex-certifai-console\n",
+    "\n",
+    "Download [certifai toolkit](https://www.cognitivescale.com/download-certifai) and follow instructions in the `Readme.md` to install the python-packages in the current environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install required certifai packages (optional packages are left for user to install)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!find $certifai_toolkit_path/packages/all       -type f ! -name \"*console-*\" | xargs -I % sh -c 'pip install % ' ;\n",
+    "!find $certifai_toolkit_path/packages/python3.6 -type f   -name \"*engine-*\"                      | xargs -I % sh -c 'pip install % ' ;\n",
+    "!find $certifai_toolkit_path/packages/all -type f   -name \"*client-*\"                      | xargs -I % sh -c 'pip install % ' ;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from certifai.client.remote import remote_config, remote_list\n",
+    "\n",
+    "class Namespace:\n",
+    "    def __init__(self, **kwargs):\n",
+    "        self.__dict__.update(kwargs)\n",
+    "\n",
+    "remote_alias = 'cpro-az'\n",
+    "pro_kubeconf_locn = 'certifai-kubeconfig.json'\n",
+    "\n",
+    "args = Namespace(file='certifai-kubeconfig.json',\n",
+    "                alias=remote_alias,\n",
+    "                context='current-context',\n",
+    "                namespace='certifai',\n",
+    "                timeout=15)\n",
+    "\n",
+    "r_conf = remote_config.remote_config(args)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### List Scan job on the Certifai Pro remote instance\n",
+    "We use the following code block to list all scan jobs on the configured `remote_alias` using the `certifai-client` python package\n",
+    "\n",
+    "\n",
+    "#### Equivalent CLI command\n",
+    "```\n",
+    "certifai remote list -a <remote_alias>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
as discussed, please use the CLI not the internal API

Reply:
fixed